Makefile

```diff
@@ -151,7 +151,8 @@ KNOWNML:=$(EXISTINGML) $(GENMLFILES) $(GENML4FILES) $(MLPACKFILES:.mlpack=.ml) \
 KNOWNOBJS:=$(KNOWNML:.ml=.cmo) $(KNOWNML:.ml=.cmx) $(KNOWNML:.ml=.cmi) \
  $(MLIFILES:.mli=.cmi) \
  $(MLLIBFILES:.mllib=.cma) $(MLLIBFILES:.mllib=.cmxa) grammar/grammar.cma
-ALIENOBJS:=$(filter-out $(KNOWNOBJS),$(EXISTINGOBJS))
+UNITTESTOBJS=$(shell find test-suite/unit-tests -name '*.cm[oi]')
```

Comment:
This is wrong, instead we should ignore the test suite in the find function (for some reason it only ignores coq-makefile)
ie this line https://github.com/coq/coq/blob/45171b6e34f01392dabee285f616f3f6f4217e98/Makefile#L61 should ignore test-suite